vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java

```diff
@@ -620,17 +622,38 @@ public Visitor(Filter filter, Map map, Point pt, boolean showOverlap) {
 
     @Override
     public Object visitDeck(Deck d) {
+      //CC// Display top n pieces of the deck instead of top piece
+
+      //if (foundPieceAt == null) {
```

Comment:
Go ahead and remove the old crud that you're replacing. It shows up in the diff when we review your PR anyway, and if PR is approved and merged we won't want the old code sitting around.